README.md

```diff
@@ -30,95 +33,152 @@ enabling greater flexibility and faster iterations.
+The default keip container provides only the basic components for Spring Integration. To fully utilize the potential of
+your integration routes, you will need to include additional Spring Integration components or your own Java code. See
 [README.md](keip-container-archetype%2FREADME.md) for instructions on how to create a custom container.
 
-Once your new container is available, you'll need to set that name in the `keip-controller-props` ConfigMap. We'll need 
+Once your new container is available, you'll need to set that name in the `keip-controller-props` ConfigMap. We'll need
 to change the value for the `integration-image` key to set it to whatever the name of your custom keip container is.
 
 ```shell
-kubectl edit configmap -n keip keip-controller-props
+kubectl -n keip edit configmap keip-controller-props
 ```
 
-Once the ConfigMap is updated, you'll need to delete the `integrationroute-webhook` container. Note that the single-line
-command below requires that you have the `jq` utility installed.
+Once the ConfigMap is updated, you'll need to restart the `integrationroute-webhook` pod.
 
 ```shell
-kubectl delete pod `kubectl get pods -n keip -o json | jq -r '.items[] | select(.metadata.name | startswith("integrationroute-webhook-")) | .metadata.name'`
+kubectl -n keip rollout restart deployment/integrationroute-webhook
 ```
 
 ### Deploying a Spring Integration Route
+
 The keip operator requires a Spring Integration route defined in XML. This XML should be stored in a ConfigMap that the
 IntegrationRoute custom resource will reference.
 
 1. Create a ConfigMap with your XML configuration:
-   ```shell
-   kubectl create configmap testroute-xml --from-file=path/to/your/route.xml
-   ```
-   
-2. Define your IntegrationRoute using the created ConfigMap:
 
-```yaml
+```shell
+cat <<'EOF' | kubectl create -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: keip-route-xml
+data:
+    integrationRoute.xml: |
+        <?xml version="1.0" encoding="UTF-8"?>
+        <beans xmlns="http://www.springframework.org/schema/beans"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xmlns:int="http://www.springframework.org/schema/integration"
+                xsi:schemaLocation="http://www.springframework.org/schema/beans
+                    https://www.springframework.org/schema/beans/spring-beans.xsd
+                    http://www.springframework.org/schema/integration
+                    https://www.springframework.org/schema/integration/spring-integration.xsd">
+            <int:channel id="output"/>
+
+            <int:inbound-channel-adapter channel="output" expression="'print me every 5 seconds'">
+              <int:poller fixed-rate="5000"/>
+            </int:inbound-channel-adapter>
+      
+            <int:logging-channel-adapter
+                  channel="output"
+                  log-full-message="true"/>
+        </beans>
+EOF
+```
```

Comment:
📓 Should we put this into examples directory to apply rather than the readme?

So readme could say something like
```
kubectl apply -f path/to/example.yaml
```

Same applies to to test route?

Reply:
On second thought should we just remove this section and refer to examples readme for how to use routes?

Reply:
We could but I wanted to have everything up-front, easy to look through, and run for the simple "quickstart" example. The examples directory provide more in-depth explanations, so I've linked those later in the doc.